### PR TITLE
update pipelines benefits

### PR DESCRIPTION
**Pull request by rosieyohannan**
The benefits displayed in the new UI, to show people why they should enable pipelines in their projects, are currently not accurate. This PR is to work on getting the text right and consistent across docs and the UI.

### CODE REVIEW

File: jekyll/_cci2/api-intro.md [hunk 1]
```diff
@@ -52,14 +52,14 @@ The CircleCI API v2 enables users to be authenticated by simply sending your API
 
 The CircleCI API v2 includes the use of pipelines to assist you in triggering workflows from the CircleCI API. By enabling pipelines, you can take advantage of specific API use cases such as:
```
Comment (rosieyohannan) [inline jekyll/_cci2/api-intro.md]:
```suggestion
The CircleCI API v2 includes the use of pipelines to assist you in triggering workflows from the CircleCI API. By enabling pipelines, you can take advantage of the following:
```

File: jekyll/_cci2/api-intro.md [hunk 2]
```diff
@@ -50,16 +50,9 @@ The CircleCI API v2 enables users to be authenticated by simply sending your API
 
 ### Pipelines
 
-The CircleCI API v2 includes the use of pipelines to assist you in triggering workflows from the CircleCI API. By enabling pipelines, you can take advantage of specific API use cases such as:
+The CircleCI API v2 includes the use of pipelines to assist you in triggering workflows from the CircleCI API. By enabling pipelines, you can take advantage of the following:
```
Comment (glenjamin) [inline jekyll/_cci2/api-intro.md]:
```suggestion
The CircleCI API v2 requires you to enable pipelines on your project. By enabling pipelines, you can also take advantage of the following:
```

Comment (rosieyohannan) [reply jekyll/_cci2/api-intro.md]: Good call!

File: jekyll/_includes/snippets/pipelines-benefits.adoc [hunk 3]
```diff
@@ -0,0 +1,10 @@
+- Use pipeline parameters to trigger [conditional workflows]({{ site.baseurl }}/2.0/pipeline-variables/#conditional-workflows).
+- Include [resuable config]({{ site.baseurl }}/2.0/reusing-config/) elements, including executors, commands, jobs, as well as packaged resuable config, known as [orbs](https://circleci.com/orbs/).
```
Comment (glenjamin) [inline jekyll/_includes/snippets/pipelines-benefits.adoc]: Perhaps something like
Access to config version 2.1, which provides:
* more detailed config validation error messages
* re-usable config
* orbs

Comment (rosieyohannan) [reply jekyll/_includes/snippets/pipelines-benefits.adoc]: Good idea

**Pull request merged**